doc/release-and-support.md

```diff
@@ -16,12 +16,12 @@ For all major and minor releases a milestone will be created at least three week
 
-The axe-core team considered security its very highest priority. While security vulnerabilities in axe-core are rare, they do happen. When they do, resolving the issue becomes are highest priority. Any commitments made prior to the discovery may be dropped.
+The axe-core team considers security its very highest priority. While security vulnerabilities in axe-core are rare, they do happen. When they do, resolving the issue becomes our highest priority. Any commitments made prior to the discovery may be dropped.
 
 ## Recommended Use of Versions
 
-In order to ensure the best quality from axe-core, we encourage everyone to regularly upgrade their version of axe-core, to try to stay as close to the latest release as possible. Depending on how axe-core is used, upgrading to a new minor or major version may result in new issues getting reported. To handle this, we recommend that you plan time to upgrade your version of axe-core at least twice a year.
+In order to ensure the best quality from axe-core, we encourage everyone to regularly upgrade their version of axe-core, to try to stay as close to the latest release as possible. Depending on how axe-core is used, upgrading to a new minor or major version may result in new issues getting reported. To handle this, we recommend that you set aside time to upgrade your version of axe-core at least twice a year.
 
-Additionally, we recommend that you always use the latest patch version of whatever minor version you are on. For example if you are using axe-core 3.5.5, and 3.5.6 is released it is best to upgrade immediately. Patch releases of axe-core should not find new issues, although they occasionally resolve issues in the case of false positives.
+Additionally, we recommend that you always use the latest patch version of whichever minor version you are on. For example if you are using axe-core 3.5.5, and 3.5.6 is released it is best to upgrade immediately. Patch releases of axe-core should not find new issues, although they occasionally resolve issues in the case of false positives.
 
-Ensuring you always use the latest available patch version of axe-core on any minor line guarantees you always the most secure version of axe-core. This minor line must have been released within the last 18 months. See [security updates](#security-updates).
+Ensuring that you always use the latest available patch version of axe-core guarantees that you are using the most secure version. This minor line must have been released within the last 18 months. See [security updates](#security-updates).
```

Comment:
The "on any minor line" part is important to keep for meaning here. If our last few releases were 4.9.2, 4.9.3, 4.10.0, 4.10.1, "the latest available patch version" means "4.10.1", but what this paragraph is saying is "even if you're still on the 4.9 minor line, you should ensure that you're using the latest patch version within that minor line (ie, 4.9.3 rather than 4.9.2)"

```suggestion
Ensuring that you always use the latest available patch version of axe-core on any minor line guarantees that you are using the most secure version. This minor line must have been released within the last 18 months. See [security updates](#security-updates).
```